astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -14,6 +14,9 @@ class CosmologyWithKwargs(Cosmology):
     def __init__(self, name="cosmology with kwargs", meta=None, **kwargs):
         super().__init__(name=name, meta=meta, **kwargs)
 
+    def equivalent(self, other):
```

Comment:
Why would this be useful?

Reply:
Since Cosmology.equivalent is an abstractmethod this definition is necessary.
But given your other comments, it might not stay abstract, in which case I will remove this here.

Reply:
Are you sure? I thought `abstractmethod` were only checked at instantiation, not class creation.

Reply:
In ``test_to_from_mapping_instance`` an instance is created by ``got = Cosmology.from_format(params, format="mapping")``.